samples/net/coap_client/src/coap-client.c

```diff
@@ -196,6 +199,26 @@ static void event_iface_up(struct net_mgmt_event_callback *cb,
 	}
 
 	k_delayed_work_init(&retransmit_work, retransmit_request);
+}
+
+static void event_iface_up(struct net_mgmt_event_callback *cb,
+			   u32_t mgmt_event, struct net_if *iface, u8_t code)
```

Comment:
type "u8_t" -> "enum coap_method"
and perhaps we should call it "method" instead of "code"?

Reply:
done